refellips/structureSE.py

```diff
@@ -352,12 +352,12 @@ class Sellmeier(ScattererSE):
 
     Parameters
     ----------
-    Am: {float, Parameter, sequence}
-        Amplitude of Sellmeier
-    En: {float, Parameter, sequence}
-        Center energy of oscillator
-    P: {float, Parameter, sequence}
-        Position of a pole (eV)
+    Am: {float, Parameter}
```

Comment:
If only all these units were in eV they would be directly comparable to Gauss/Lorentz.